docs/release-notes/release-notes-0.9.9.md

```diff
@@ -0,0 +1,155 @@
+
+### Fixes to sendpayment
+
+Two bugs in sendpayment were fixed:
+
+Allowing sub-27K sats for non-coinjoin payments (our `DUST_THRESHOLD` is not "the" Bitcoin dust threshold, it was deliberately set higher to account for certain estimation uncertainties related to coinjoin; but they are irrelevant for non-coinjoins):
+* `d6d40df` Allow amounts below DUST_THRESHOLD (0.00027300 BTC) for non-cj direct sends
+
+Fixes a bug where sending to a BIP21 URI fails (bug as described [here](https://github.com/JoinMarket-Org/joinmarket-clientserver/issues/1356#issue-1384971769)).
+* `9d9035b` Fix sendpayment without amount argument, BIP21 only
+
+### Switch to venv from virtualenv
+
+`venv` is provided as standard in all versions of Python that we support, so we don't actually need the dependency `virtualenv`. This removes it; there is no impact at the user level.
+* `11ddec7` docs: Remove now-confusing mentions of virtualenv
+* `6e3b6ec` Dockerfiles: Use Python venv instead of virtualenv
+* `8bbca60` install: Use python3's venv module when available
+
+### Bugfixes and minor changes
+
+This fix prevents directory nodes crashing in certain edge cases:
+* `bfb0e25` Account for missing nick fields in OnionPeer
+
+This fix ensures fee randomness works even when a fee estimate can't be sourced from Core:
+* `3d0f2d6` fix: randomize fallback transaction fee
+
+* `2e44f00` Log IRC connection attempts when using SOCKS5 proxy or TLS too
+
+* `2749da1` change payjoin default binding port
+
+This was a rare to trigger, but pernicious bug: a user's wallet could show incorrect balances, if the RPC calls to the blockchain failed (the code in `sync_unspent` was incorrectly ignoring this failure), this is no longer allowed:
+* `0b34e0b` Abort sync_unspent if blockheight RPC call fails
+
+### Documentation
+
+* `50e3196` Update onion-message-channels.md
+* `44aea32` Fix typo in `gettimelockaddress`
+* `b34b932` Correction to docs: no coinjoin spends of FBs
+* `4c53bb5` Update link to RaspiBolt installation guide
+
+### Installation and dependencies
+
+* `63d74bc` Call num_cores() only after deps_install()
+* `7d0ca22` Update local Tor to 0.4.7.12
+* `b59fdcd` Remove Python 3.6 (EOL), add 3.10
+* `01dc08a` Update local Tor to 0.4.7.13
+* `e47f082` Allow to run install.sh from any current working directory
+* `46013d2` chmod +x scripts/jmwalletd.py
+* `8a25e3a` Pin remaining deps to specific versions, except for Qt and tests
+* `22c13b0` Remove unused code instead of commenting out
+
+Mostly applies to the shell scripts used for installation, hence included here:
+* `b928713` Add ShellCheck linter script
+* `4f0eebc` Apply all current shellcheck suggestions to rest of the scripts
+* `f0b9872` Apply all current shellcheck suggestions to install.sh
+
+* `bd5508c` Minor improvements for question prompts and feedback
+
+* `890dd50` Bump pytest version to 6.2.5
```

Comment:
Also "Testing", pytest is not installed for ordinary users.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1443/commits/978263b4e72edb18c47c164ce609baa63932da39